pkg/metrics/object.go

```diff
@@ -254,3 +270,15 @@ func (m objectServiceMetrics) AddPutPayload(ln int) {
 func (m objectServiceMetrics) AddGetPayload(ln int) {
 	m.getPayload.Add(float64(ln))
 }
+
+func (m objectServiceMetrics) ChangeBy(shardID string, delta int) {
+	m.shardMetrics.With(prometheus.Labels{shardIDLabelKey: shardID}).Add(float64(delta))
+}
+
+func (m objectServiceMetrics) IncObjectCounter(shardID string) {
+	m.shardMetrics.With(prometheus.Labels{shardIDLabelKey: shardID}).Inc()
+}
+
+func (m objectServiceMetrics) DecObjectCounter(shardID string) {
+	m.shardMetrics.With(prometheus.Labels{shardIDLabelKey: shardID}).Dec()
+}
```

Comment:
Why did you decide to stay on 3 methods instead of a single `AddToObjectCounter` which covers both `inc` and `dec`?

Reply:
kept only two: `Set*` and `Add*`